metricflow/plan_conversion/dataflow_to_sql.py

```diff
@@ -915,8 +916,10 @@ def visit_join_aggregated_measures_by_groupby_columns_node(  # noqa: D
                         column_aliases=join_aliases,
                         left_source_alias=from_data_set_alias,
                         right_source_alias=right_data_set_alias,
-                    ),
-                    join_type=SqlJoinType.INNER,
+                    )
+                    if not use_cross_join
+                    else None,
```

Comment:
Oof, this is hard to read, but don't worry about it - I'm refactoring all of these join helpers separately so I'll deal with it when I get to this node type.